src/TuDa.CIMS.Web/Components/SubmitButton.razor.cs

```diff
@@ -0,0 +1,20 @@
+using MudBlazor;
+
+namespace TuDa.CIMS.Web.Components;
+
+public partial class SubmitButton
+{
+    /// <summary>
+    /// DialogOption to implement the close button.
+    /// </summary>
+    private readonly DialogOptions _closeButton = new() {CloseButton = true};
```

Comment:
⛏️ You could remove this and just insert the value as parameter in the `OpenDialog` method